bundle-size/comment.js

```diff
@@ -4,14 +4,44 @@
 /** @typedef {import('./compareStats').DiffResult} DiffResult */
 /** @typedef {import('./compareStats').EntryDiffResult} EntryDiffResult */
 
-const prMessageSymbol = `<!-- grafana-plugin-actions-bundle-size-comment -->`;
```

Comment:
In IRM monorepo we need symbol per pnpm workspace so that for each workspace we get a new comment, see `getPrMessageSymbol` function below